pages/common/lwp-request.md

```diff
@@ -0,0 +1,27 @@
+# lwp-request
+
+> Simple Perl-based command-line HTTP client.
```

Comment:
libwww-perl isn't terribly useful info, though maybe it's more useful than "Perl". The reason I think I mentioned Perl in the first place is because, while lwp-request is running, it sets your terminal title to "perl5.18" (or whatever version of Perl you have), so I tend to think of it as "that perl-based http tool".